grid_size:
  row: 4
  col: 3
  len: 1.5
policy:
  policy_file: "/home/redwan/CLionProjects/T-ASE/result/policy.txt"
  belief_file: "/home/redwan/CLionProjects/T-ASE/result/belief.txt"
  test_file: "/home/redwan/CLionProjects/T-ASE/config/wafr_tests/test_instances/p_3/corridor_test_11"
topic_pub:
  vicon: "/vicon/fetch/fetch"
  goal: "/fetch_move_base_planner/query_goal"
  repeat: 3
move_base:
  xy_tolarance: 0.265
  err_viz: 0.3
  nap_rate: 10
